R/difftime.R

```diff
@@ -1,6 +1,10 @@
 
 renv_difftime_format <- function(time, digits = 2L) {
 
+  if (is_testing()) {
```

Comment:
Are you ok with leaving this as is until I do a patch release of testthat? (Hopefully early next week)

Reply:
I'm okay with that; should we add an issue or some other reminder to do so?

Reply:
I'll make a note on my personal todo list.